crates/viewer/re_space_view_map/src/visualizers/geo_points.rs

```diff
@@ -0,0 +1,163 @@
+use re_space_view::{DataResultQuery as _, RangeResultsExt as _};
+use re_types::{
+    archetypes::GeoPoints,
+    components::{Color, LatLon, Radius},
+    Loggable as _,
+};
+use re_viewer_context::{
+    auto_color_for_entity_path, IdentifiedViewSystem, QueryContext, SpaceViewSystemExecutionError,
+    TypedComponentFallbackProvider, ViewContext, ViewContextCollection, ViewQuery,
+    VisualizerQueryInfo, VisualizerSystem,
+};
+
+// ---
+
+#[derive(Debug, Clone)]
+struct GeoPointEntry {
+    /// Position.
+    position: walkers::Position,
+
+    /// Display radius in pixels
+    //TODO(#7872): support for radius in meter
+    radius: f32,
+
+    /// Color.
+    color: egui::Color32,
+}
+
+/// A map scene, with entries on the map to render.
+#[derive(Default)]
+pub struct GeoPointsVisualizer {
+    map_entries: Vec<GeoPointEntry>,
+}
+
+impl IdentifiedViewSystem for GeoPointsVisualizer {
+    fn identifier() -> re_viewer_context::ViewSystemIdentifier {
+        "GeoPoints".into()
+    }
+}
+
+impl VisualizerSystem for GeoPointsVisualizer {
+    fn visualizer_query_info(&self) -> VisualizerQueryInfo {
+        VisualizerQueryInfo::from_archetype::<GeoPoints>()
+    }
+
+    fn execute(
+        &mut self,
+        ctx: &ViewContext<'_>,
+        view_query: &ViewQuery<'_>,
+        _context_systems: &ViewContextCollection,
+    ) -> Result<Vec<re_renderer::QueueableDrawData>, SpaceViewSystemExecutionError> {
+        for data_result in view_query.iter_visible_data_results(ctx, Self::identifier()) {
+            let results = data_result.query_archetype_with_history::<GeoPoints>(ctx, view_query);
+
+            // gather all relevant chunks
+            let timeline = view_query.timeline;
+            let all_positions = results.iter_as(timeline, LatLon::name());
+            let all_colors = results.iter_as(timeline, Color::name());
+            let all_radii = results.iter_as(timeline, Radius::name());
+
+            // default component values
```

Comment:
nit: convolutes "default" & "fallback". Something I'm very guilty of as well by making fallbacks use defaults by default, but they are distinct things since fallbacks may be a lot more complex than Rust defaults